src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -892,14 +949,64 @@ public DescriptorImpl getDescriptor() {
     private static class GroupDetailsImpl extends GroupDetails {
 
         private String name;
+        private final Set<String> members;
 
         public GroupDetailsImpl(String name) {
             this.name = name;
+            this.members = null;
+        }
+
+        public GroupDetailsImpl(String name, Set<String> members) {
+            this.name = name;
+            this.members = members;
         }
 
         public String getName() {
             return name;
         }
+
+        @Override
+        public Set<String> getMembers() {
+            return members;
+        }
+    }
+
+    private static class LdapGroupDetailsMapper implements LdapEntryMapper {
+        @Override
+        public Object mapAttributes(String dn, Attributes attributes) throws NamingException {
+            String cn = fixGroupname(String.valueOf(attributes.get("cn").get()));
+            NamingEnumeration<?> enumeration;
+            boolean expectingUidInsteadOfDn = false;
+            if (attributes.get("member") != null) {
```

Comment:
Hardcoded assumption on one specific implementation of `LDAPGroupMembershipStrategy`